lib/protobuf/generators/enum_generator.rb

```diff
@@ -32,6 +32,7 @@ def compile
 
       def build_value(enum_value_descriptor)
         name = enum_value_descriptor.name
+        name.upcase! if ENV.key?('PB_ENUM_UPCASE')
```

Comment:
Thank you! One change recommendation and I'll merge this. Can we change this from `PB_ENUM_UPCASE ` to `PB_UPCASE_ENUMS`? That way it fits with the language of `PB_IGNORE_DEPRECATIONS` and other env variables.